tests/test_ag3.py

```diff
@@ -375,7 +375,7 @@ def test_snp_sites_for_joined_arms(chrom):
     assert isinstance(sites_actual, da.Array)
     assert sites_actual.ndim == 1
     assert sites_actual.dtype == "int32"
-    assert np.all(sites_expected == sites_actual)
+    assert da.all(sites_expected == sites_actual)
```

Comment:
This won't actually test anything I don't think, because return value of da.all() is an uncomputed dask array. To test it you'd need to `assert da.all(sites_expected == sites_actual).compute()`

Reply:
If you're still getting blosc errors, can try:

```
assert da.all(sites_expected == sites_actual).compute(scheduler='single-threaded')
```

...IIRC we had to do something similar for testing the genome sequence method.

Reply:
Thanks! I've now added this in the #355 PR